gc3libs/config.py

```diff
@@ -308,7 +325,8 @@ def merge_file(self, filename):
             "Configuration.merge_file(): Reading file '%s' ...",
             filename)
         with open(filename, 'r') as stream:
-            defaults, resources, auths = self._parse(stream, filename)
+            parser = self.read_file(stream, filename)
```

Comment:
I'm not sure I like the split of `self._parse()` into methods named `read_file()` (which parses the file) and `_parse()` (which separates the parsed values into dictionaries `resources`, `auths`, and `defaults`).

How about using `_parse()` and `_split()` instead?

Reply:
(I know this sounds very nit-picky, but keeping function names close to what the code really does is important for readability in the long run, IMHO.)

Reply:
How does e1b424498c227249f3c43f80cca6a19cd2d1be61 look?

Reply:
:+1: